en/docs/sdks/start-integrating-apps/integrate-a-react-app.md

```diff
@@ -54,6 +54,20 @@ To add CORS configurations:
     !!! note
         Make a note of the **OAuth Client Key** and **OAuth Client Secret** that appear, as they will be used to configure the sample application.
 
+## Configure the Resident Identity Provider
+
+!!! Note
+    Follow this step, only if you are running the WSO2 Identity Server on a custom domain/port.
+
+1. On the WSO2 Identity Server management console (`https://<IS_HOST>:<PORT>/carbon`), go to **Main** > **Identity** > **Identity Providers**.
+2. Click **Resident** and expand **Inbound Authentication Configuration**.
+3. Expand **OAuth2 / OpenID Connect Configuration** and update **Identity Provider Entity ID** value to match with your custom domain/port.
+
+    !!! info
+        For example, if you are running the WSO2 Identity Server on port 9500, the updated **Identity Provider Entity ID** value should be `https://localhost:9500/oauth2/token`.
```

Comment:
Lets remove this info box and add it as a usual text within the `!!! Note`.

The reason, we don't usually add an info box inside another info box.

Reply:
> we don't usually add an info box inside another info box.

I haven't though :)

anyways, I removed the note and formatted the section. Please see the attached ss in PR description.

Reply:
> we don't usually add an info box inside another info box.

I haven't though :)

anyways, I removed the note and formatted the section. Please see the attached ss in PR description.